src/components/views/messages/SenderProfile.js

```diff
@@ -125,8 +125,10 @@ export default createReactClass({
         </span>;
 
         const content = this.props.text ?
-            <span className="mx_SenderProfile_aux">
-                { _t(this.props.text, { senderName: () => nameElem }) }
+            <span>
```

Comment:
What's the point of this extra unnamed span...?

Reply:
For some reason I couldn't figure out how to make the css work, I sunk a bunch of time into it. I think it's because the surrounding span is a flex, but making this the same structure as the mx_SenderProfile_name worked. It's a bit of a hack

Reply:
Okay, seems fine enough then, as along Design is okay as well.